includes/class-wc-bfx-pay-gateway.php

```diff
@@ -184,6 +184,51 @@ protected function update_order_status($order, $payment_status)
         return false;
     }
 
+    protected function bfx_request($path, $body) {
```

Comment:
can we reuse this also on `process_payment` please?

Reply:
I have updated it. Please check! thanks!